app/src/main/java/com/duckduckgo/app/tabs/model/TabRepository.kt

```diff
@@ -66,7 +66,7 @@ interface TabRepository {
      */
     suspend fun purgeDeletableTabs()
 
-    suspend fun deleteCurrentTabAndSelectSource()
+    suspend fun deleteCurrentTabAndSelectSource(tabId: String)
```

Comment:
Same here, I would remove the `currentTab` reference from the method name since we can pass any tab id as parameter.

Reply:
👍